docs/src/main/asciidoc/tests-with-coverage.adoc

```diff
@@ -173,6 +174,12 @@ Now we need to add Jacoco to our project. To do this we need to add the followin
 </dependency>
 ----
 
+.Gradle
```

Comment:
I would more add a sentence like `If you are using Gradle, add the following dependency to our `build.gradle` file: `